### ISSUE TITLE
Working 50%

### ISSUE THREAD

**Issue by jjmcousineau** [2dom/PixelTimes#3]
Followed guide to the letter, great instructions by the way. Test pattern works as described but pixel times only displays on rows 0-3 and  8-11. I tried to find any settings that may differ between the pattern and the pixeltimes program but I could not find it. 

I would greatly appreciate help with this

-edit: I realized I had the panel upside down (wiring was unaffected) so the row numbers were off.

**Comment by 2dom** [comment]
Hi,

sounds like the one of the address pins is stuck. Please re-check your connection for your A,B,C pins (in particular C). Maybe also try another pin.

**Comment by jjmcousineau** [comment]
Hey Thanks for the reply I was able to get it working after all. It was more of an RFTM problem than anything. The current commit has the Row Step set to 4 and not 8 like in the example files (display.begin(4)). Once I figured out how the panels really worked it was obvious that the issue was in setup in the program. I am happy that it was that way as it forced me to learn how it works better than if I had just been able to plug and play.

Thanks again for all the hard work.
